examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.cpp

```diff
@@ -605,11 +598,13 @@ sl_status_t bg_scan_callback_handler(sl_wifi_event_t event, sl_wifi_scan_result_
 static void wfx_rsi_save_ap_info() // translation
 {
     sl_status_t status = SL_STATUS_OK;
-#ifndef EXP_BOARD // TODO: this changes will be reverted back after the SDK team fix the scan API
+#ifndef EXP_BOARD
+    // TODO: this changes will be reverted back after the SDK team fix the scan API
```

Comment:
What needs to be fixed by the SDK team? Which SDK team (Matter SDK or Silabs SDK?)